src/us/kbase/genbank/test/TestCommandLine.java

```diff
@@ -25,7 +25,7 @@
  * Date: 10/29/14
  * Time: 12:17 AM
  */
-public class CommandLineTest {
+public class TestCommandLine {
```

Comment:
That makes sense.  I'm going to give this particular one a pass for now,
but we do need to settle on some of these coding conventions generally and
have them be consistent across the project.

On Tue, Apr 14, 2015 at 3:38 PM, jmchandonia <notifications@github.com>
wrote:

> In src/us/kbase/genbank/test/TestCommandLine.java
> <https://github.com/kbase/transform/pull/146#discussion_r28378163>:
>
> > @@ -25,7 +25,7 @@
> >   * Date: 10/29/14
> >   * Time: 12:17 AM
> >   */
> > -public class CommandLineTest {
> > +public class TestCommandLine {
>
> I renamed @realmarcin <https://github.com/realmarcin>'s class so it
> wouldn't look like a class with JUnit unit tests. Here
> and elsewhere in KBase, those classes are in a test/ directory and also
> named
>
>
> *Test.java. This seems to be the most popular convention:
> http://stackoverflow.com/questions/3146821/naming-convention-junit-suffix-or-prefix-test
> <http://stackoverflow.com/questions/3146821/naming-convention-junit-suffix-or-prefix-test>
> Besides FormatTest, which I added, I don't know what the other test classes
> do, or even if they contain any code that still tests anything useful. But
> none of the others contain unit tests, and most seem to follow a Test*.java
> naming convention, so I renamed this one for consistency.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kbase/transform/pull/146/files#r28378163>.
>